UnityProject/Assets/StreamingAssets/admin/permissions.toml

```diff
@@ -0,0 +1,29 @@
+﻿[ranks.host]
+show_in_chat = false
+abbreviation = "H"
+color = "red"
+permissions = ["perma_promote", "promote", "demote", "kick", "ban", "fun"]
+
+[ranks.admin]
+show_in_chat = true
+abbreviation = "A"
+color = "blue"
+permissions = ["promote", "kick", "ban", "fun"]
+
+[ranks.normal]
+show_in_chat = false
+abbreviation = "N"
+color = "white"
+permissions = []
+
+[[players]]
+identifier = "hostPlayer"
+rank = "host"
+
+[[players]]
+identifier = "adminPlayer"
+rank = "admin"
+
+[[players]]
+identifier = "normalPlayer"
+rank = "normal"
```

Comment:
Maybe add an example comment of how to add a user to one of these?

Reply:
In this file you create the ranks and assign them to users.

To add a new record you just add the [[players]] list and under it, the identifier (ugly ass firebase string) and the rank you created above.

I thought it was self explanatory.

Reply:
```suggestion
[[players]]
identifier = "hostPlayerUserID"
rank = "host"

[[players]]
identifier = "adminPlayerUserID"
rank = "admin"

[[players]]
identifier = "normalPlayerUserID"
rank = "normal"
```

Maybe that then?

Reply:
Sure, the ranks I created there are merely for testing purposes. I didn't intend to do the actual ranks we will have nor the permissions.

The normal user on the bottom only represents a record without permissions. Of course we don't need it in the game.

Reply:
I separated the example config file form the test one. Now the example is a better representation of reality. Take a look.

Reply:
Yeah, i think that explains it well.